inference/core/managers/decorators/fixed_size_cache.py

```diff
@@ -90,6 +90,22 @@ async def infer_from_request(
         self._key_queue.append(model_id)
         return await super().infer_from_request(model_id, request, **kwargs)
 
+    def infer_from_request_sync(
```

Comment:
If we are moving away from async, maybe we don't need to signify this method is non-async - lack of `async` in method definition signifies that

Reply:
we do not yet - this is my long term plan, but currently we have both symbols:
* `def infer_from_request_sync()`
* `async def def infer_from_request()`